server/models/images.js

```diff
@@ -51,7 +51,7 @@ module.exports.updateEntry = (imageToUpdate, callback) => {
   // const queryStr = 'SELECT * FROM `images` WHERE hostel_id = ?';
   // const queryArgs = [id];
   const queryStr = 'UPDATE `images` SET `file_name` = ?, `url` = ?, `description` = ?, `hostel_id` = ? WHERE `id` = ?';
-  const queryArgs = [imageToUpdate.fileName, imageToUpdate.url, imageToUpdate.description, imageToUpdate.hostelId, imageToUpdate.id];
+  const queryArgs = [imageToUpdate.file_name, imageToUpdate.url, imageToUpdate.description, imageToUpdate.hostel_id, imageToUpdate.id];
```

Comment:
Even more so, now that the keys of the object are what they should be for the database you should juts pas an object into the query args